frontend/src/views/ProducerFormPage/SummaryStep.vue

```diff
@@ -103,6 +103,16 @@ const conditionNames = computed(() => {
 })
 
 const editLink = (step) => ({ name: "ProducerFormPage", query: { step } })
+
+const saveDraft = async () => {
+  const { response } = await useFetch("/api/v1/declarations/", { headers: headers() }).post(payload)
+  if (response.value.ok) {
+    await router.replace({ name: "DashboardPage" })
+    useToaster().addSuccessMessage("Votre démarche a été sauvegardée")
+  } else {
+    console.log(response)
```

Comment:
Est-ce qu'il y a une raison de ne pas appeler notre util `handleError` pour gérer le cas où le endpoint échoue ?

Reply:
Non, c'est juste out-of-scope de la PR (un peu lié à la conversation de la gestion UI d'erreurs sur cette page. Ça arrivera par la suite).